src/config.cpp

```diff
@@ -655,29 +694,69 @@ class CircuitConfig::Parser
         result.biophysicalNeuronModelsDir = getJSONPath(components,
                                                         "biophysical_neuron_models_dir");
 
+        result.vasculatureFile = getOptionalJSONPath(components, "vasculature_file");
+        result.vasculatureMesh = getOptionalJSONPath(components, "vasculature_mesh");
+        result.endfeetMeshesFile = getOptionalJSONPath(components, "endfeetMeshesFile");
```

Comment:
Why is this in camel case, while the rest in snake case?

Reply:
tnx, fixed